course_discovery/apps/api/serializers.py

```diff
@@ -1915,388 +1882,6 @@ def get_narrow_url(self, instance):
         return serializers.Hyperlink(url, 'narrow-url')
 
 
-class BaseHaystackFacetSerializer(HaystackFacetSerializer):
```

Comment:
All this removed code is what you are replacing with the `search_index` djangoapp with its DSL approach, yah?

Reply:
Yes exactly. I developed serializers based on django-es-dsl-drf and put them in a separate `search_index` package. There are serializers that handle responses from `elasticsearch`. This refactoring seemed to be correct and necessary. It's in the spirit of python. And now the `serializers.py` module itself is lighter and smaller and easier to navigate and figure out.